app/graphql/types/submission_type.rb

```diff
@@ -8,7 +8,7 @@ class SubmissionType < Types::BaseObject
     field :artist_id, String, null: false
     field :assets, [Types::AssetType, null: true], null: true
     field :authenticity_certificate, Boolean, null: true
-    field :category, Types::CategoryType, null: true
+    field :category, String, null: true
```

Comment:
@jonallured - i believe these things are connected: 

![image](https://user-images.githubusercontent.com/236943/78054598-2d61c880-7337-11ea-8c80-634663dead08.png)

Reply:
@damassi we can specify the type of the input to the mutation separately from the type of the return from the query. They are not connected in that way.

Reply:
Sounds good, how all these things connect is still a bit labrythine to me so wanted to surface just in case